frontend/dispatcher.lua

```diff
@@ -88,8 +88,8 @@ local settingsList = {
     screenshot = { category="none", event="Screenshot", title=_("Screenshot"), device=true, separator=true,},
 
     -- filemanager settings
-    folder_up = { category="none", event="FolderUp", title=_("Folder up"), filemanager=true},
-    show_plus_menu = { category="none", event="ShowPlusMenu", title=_("Show plus menu"), filemanager=true},
+    folder_up = { category="none", event="FolderUp", title=_("Folder up"), filemanager=true,},
+    show_plus_menu = { category="none", event="ShowPlusMenu", title=_("Show plus menu"), filemanager=true,},
     folder_shortcuts = { category="none", event="ShowFolderShortcutsDialog", title=_("Folder shortcuts"), filemanager=true, separator=true,},
```

Comment:
Can't say I'm a big fan of the way that looks. It's not really relevant to why we usually prefer a trailing comma.

Reply:
Ok, then we should remove the trailing commata, so the code style is consistent?
Right now (`true},` and `true,}`) it looks like am mess.

Reply:
Oh, in that sense. I thought you were changing everything because our style guide says we prefer trailing commas. Which we do... in a very different scenario. Let's await some more opinions.

Reply:
Trailing commas have sense on multi-line tables, so adding a new item does not generate a diff on the other lines/items.
Here, on a fully single-line table, it does not make much sense. To me, it looks cleaner without a trailing comma. (And if we need to add an item, we'll add the comma to the line we have to change anyway.)